src/componentsNext/FilePicker/FilepickerItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, Fragment } from "react";
+import {
+  FilepickerClose,
+  FilepickerListItem,
+  FilepickerSeparator,
+} from "~/design/Filepicker";
+
+import { NodeSchemaType } from "~/api/tree/schema/node";
+import { fileTypeToIcon } from "~/api/tree/utils";
+import { twMergeClsx } from "~/util/helpers";
+
+export type FilepickerItemType = {
+  results: NodeSchemaType[];
+  selectable: ((node: NodeSchemaType) => boolean) | undefined;
+  setPath: (path: string) => void;
+  setInputValue: (value: string) => void;
+  onChange: (path: string) => void;
+};
+
+export const FilepickerItem: FC<FilepickerItemType> = ({
```

Comment:
Should we rename this to `FileList`? My initial thought when reading the name was that this renders one item.